src/mumble/GlobalShortcut_xdp.cpp

```diff
@@ -0,0 +1,200 @@
+// Copyright 2022 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "GlobalShortcut_xdp.h"
+
+#include "EnvUtils.h"
+#include "Global.h"
+#include "GlobalShortcutTypes.h"
+#include "Settings.h"
+
+#include <QWindow>
+
+#include "globalshortcuts_portal_interface.h"
+#include "portalsrequest_interface.h"
+
+Q_GLOBAL_STATIC_WITH_ARGS(OrgFreedesktopPortalGlobalShortcutsInterface, s_shortcutsInterface, (QLatin1String("org.freedesktop.portal.Desktop"),
+                                                                        QLatin1String("/org/freedesktop/portal/desktop"),
```

Comment:
```suggestion
Q_GLOBAL_STATIC_WITH_ARGS(OrgFreedesktopPortalGlobalShortcutsInterface, s_shortcutsInterface, ("org.freedesktop.portal.Desktop",
                                                                        "/org/freedesktop/portal/desktop",
```
(For Stringliterals we no longer require `QLatin1String` wrappers - this has changed with the introduction of cmake and therefore a lot of old code still uses it)